### PR TITLE
Update community sponsor and blog card styles

### PR DESCRIPTION
**Pull request by tessaSAC**
**Made following edits under @reshamas guidance:**
- Add website link to community sponsor images
- Add community name linked to website
- Add community location
- Add Twitter icon next to Twitter links
- Make Twitter links display handle rather than url
- Make Twitter links conditional based on whether there is a Twitter link
- Clean up text alignment of blog cards

### CODE REVIEW

File: pages/blog.vue [hunk 1]
```diff
@@ -75,10 +75,13 @@
         margin: 20px 0px;
     }
     .card-block {
+        padding: 0 14px;
         height: 200px;
         overflow: hidden;
     }
-    .card-text {
+    .card-text.card-text {
```
Comment (tessaSAC) [inline pages/blog.vue:82]: duplicate class enables this to override `.card-text` styles which appear to be reset elsewhere

File: pages/sponsors.vue [hunk 2]
```diff
@@ -63,10 +63,21 @@
...
+                                </a>
+                            </div>
                             <div class="cell auto sponsor-text-plain">
+                                <a :href="sponsor.website" target="_blank">
+                                    <h2>{{ sponsor.name }}</h2>
+                                </a>
+                                <p>{{ sponsor.location }}</p>
                                 <p>
-                                    <a :href="sponsor.link">{{ sponsor.link }}</a>
+                                    <a v-if="sponsor.twitterLink" :href="sponsor.twitterLink" target="_blank">
```
Comment (tessaSAC) [inline pages/sponsors.vue:77]: For the other link you probably want a `<a v-else :href="sponsor.otherLink">` or `<a v-else-if="predicate" :href="sponsor.otherLink">`

File: pages/sponsors.vue [hunk 3]
```diff
@@ -225,4 +253,10 @@
     .sponsor-text-plain {
         margin: 30px;
     }
+
+    .icon-twitter {
+        height: 20px;
+        margin-right: 4px;
+        margin-bottom: 1px;
```
Comment (tessaSAC) [inline pages/sponsors.vue:260]: margin-bottom is quick hack to make icon look more vertically centered

File: pages/sponsors.vue [hunk 4]
```diff
@@ -104,78 +115,95 @@
...
-                        link: "https://twitter.com/ChAnalyticsPyth",
+                        twitterLink: "https://twitter.com/ChAnalyticsPyth",
+                        twitterHandle: "@ChAnalyticsPyth",
                         description: "Mayor comunidad en Chile de Analítica y Python. Espacio para compartir conocimiento y aprender.",
                         image: require("~/assets/images/sponsors/analytics_python.png")
                     },
                     {
-                        link: "https://womeninanalytics.com",
+                        twitterLink: "https://womeninanalytics.com",
+                        twitterHandle: "@",
```
Comment (SylvainCorlay) [inline pages/sponsors.vue:152]: Is it @wia_community?

File: pages/sponsors.vue [hunk 5]
```diff
@@ -104,78 +115,95 @@
...
                         description: "Women in Analytics is a global organization that brings together an international community of industry leaders, technical experts, entrepreneurs, and academia. Our mission is to provide visibility to the women making an impact in the analytics space and furnish a platform for them to lead the conversation around the advancements of analytical research, development, and application. WIA aims to be an extremely inclusive organization and welcomes participation from all genders.",
                         image: require("~/assets/images/sponsors/WIA-Secondary-RGB.png")
                     },
                     {
-                        link: "https://berlin.pydata.org/",
+                        twitterLink: "https://berlin.pydata.org/",
+                        twitterHandle: "@",
```
Comment (SylvainCorlay) [inline pages/sponsors.vue:158]: @pydataberlin

**Comment by SylvainCorlay** [comment]
Many thanks! I left a few review comments online (mostly adding new twitter handles).

**Comment by reshamas** [comment]
@SylvainCorlay I can go back in and add in all the twitter handles (and names).  I was hoping this PR would be accepted, and then I can add in the additional info.  @tessaSAC set it up for me in a way that if a group doesn't have a twitter handle, it's ok, the link doesn't show up.

**Comment by SylvainCorlay** [comment]
OK, sounds good.

Merging and we can have another PR with the twitter handles.

**Pull request merged**